index.js

```diff
@@ -30,7 +30,6 @@ function sendBootStatus( status ) {
 }
 
 console.log( chalk.yellow( '%s booted in %dms - http://%s:%s' ), pkg.name, ( Date.now() ) - start, host, port );
-console.info( chalk.cyan( '\nGetting bundles ready, hold on...' ) );
```

Comment:
This bothered me a lot. In a non-development build, you *don't* have to wait for anything, but this message would still be displayed. I waited for a good 20 minutes like an idiot 😅